### PR TITLE
Docs reorg

### PR DESCRIPTION
**Pull request by amn41**
**Proposed changes**:
- use new docs theme
- separate 'user guide' and 'getting started' sections
- push built docs to 'newdocs' branch

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by amn41** [comment]
@tmbo do you want to review this or assign someone?

File: rasa_nlu/model.py [hunk 1]
```diff
@@ -60,7 +60,14 @@ class Metadata(object):
     @staticmethod
     def load(model_dir):
         # type: (Text) -> 'Metadata'
-        """Loads the metadata from a models directory."""
+        """
+        Loads the metadata from a models directory.
```
Comment (tmbo) [inline rasa_nlu/model.py]: docs convention is to put the summary line right after `"""`. e.g.
```
"""Loads the metadata from a models directory.

Args: ...
```

File: rasa_nlu/config.py [hunk 2]
```diff
@@ -93,7 +93,10 @@ class RasaNLUModelConfig(object):
     DEFAULT_PROJECT_NAME = "default"
 
     def __init__(self, configuration_values=None):
-
+        """
+        Create a model configuration, optionally overridding
```
Comment (tmbo) [inline rasa_nlu/config.py]: summary line should be on same line as `"""` and should be a single line.

File: rasa_nlu/model.py [hunk 3]
```diff
@@ -263,12 +270,21 @@ def ensure_model_compatibility(metadata):
 
     @staticmethod
     def load(model_dir, component_builder=None, skip_validation=False):
-        """Creates an interpreter based on a persisted model."""
+        """
+        Create an interpreter based on a persisted model.
```
Comment (tmbo) [inline rasa_nlu/model.py]: same

**Pull request merged**